### PR TITLE
End to end multibranch

### LINKED ISSUE #38: feature(Tutorial 1): End-to-End Multibranch Pipeline Project Creation
- For discussion to replicate Tutorial [End-to-End Multibranch Pipeline Project Creation](https://www.jenkins.io/doc/tutorials/build-a-multibranch-pipeline-project/)
- This tutorial has a Blueocean section, which we can replace with [Pipeline graph view plugin](https://plugins.jenkins.io/pipeline-graph-view/)
- Uses  docker in docker 
- Pinging mentors @berviantoleo @gounthar @jmMeessen

### PR DESCRIPTION
**Pull request by ash-sxn**
- fixes #38 
- In This tutorial I've included blueocean plugin since it was used extensively
- using the same agent as node Tutorial worked fine since it's based on same application
- I've Updated the Readme file according to the [Tutorial](https://www.jenkins.io/doc/tutorials/build-a-multibranch-pipeline-project/)  
- Everything is working other than when the production branch is ran it shows website on port `3000` instead of port `5000`, which I don't understand why right now.

### CODE REVIEW

**Comment by gounthar** [comment]
There is still an error with the tests, right?

File: docker-compose.yaml [hunk 1]
```diff
@@ -103,6 +103,49 @@ services:
       retries: 5
     volumes:
       - agent-ssh-dir:/home/jenkins/.ssh:ro  # Mounts the agent-ssh-dir volume to the /home/jenkins/.ssh path inside the container as read-only
+
+
+  multi_jenkins_controller:
+    image: ${DOCKERHUB_USERNAME}/jenkinsci-tutorials:multi_controller_end-to-end-multibranch
```
Comment (berviantoleo) [inline docker-compose.yaml:109]:
```suggestion
    image: ${DOCKERHUB_USERNAME}/jenkinsci-tutorials:multi_controller_end-to-end-multibranch_
```

I think, currently have _ in the end, right?

Comment (berviantoleo) [inline docker-compose.yaml:109]:
```suggestion
    image: ${DOCKERHUB_USERNAME}/jenkinsci-tutorials:multi_controller_
```

I just want to ensure, do we need the branch? (Especially after merging)

Comment (ash-sxn) [reply docker-compose.yaml:109]: We won't need it after merging, but it's needed right now,
Should I remove it?

Comment (gounthar) [reply docker-compose.yaml:109]: I guess so! 👍

**Review by berviantoleo — APPROVED** [review]

**Pull request merged**